templates/etc/sysconfig/docker.systemd.erb

```diff
@@ -9,6 +9,7 @@ OPTIONS="<% if @root_dir %> -g <%= @root_dir %><% end -%>
  --iptables=<%= @iptables -%>
  --ip-masq=<%= @ip_masq -%>
 <% unless @icc == nil %> --icc=<%= @icc %><% end -%>
+<% if @registry_mirror != 'undef' %> --registry-mirror=<%= @registry_mirror %><% end -%>
```

Comment:
@MWilsonPuppet we can use the same pattern as the other params here. We only need to test if registry_mirror is defined, not that it's contents doesn't equal the string 'undef'. 

so I'm thinking something like this:

```
<% if @registry_mirror %> --registry-mirror=<%= @registry_mirror %><% end -%>
```

Then change the default value of registry mirror, back to undef, not 'undef'

Reply:
@davejrt Ive made the requested changes and  acceptance and unit tests are passing.
cheers
Mark